tox/session.py

```diff
@@ -298,13 +299,16 @@ def good(self, msg):
         self.logline(msg, green=True)
 
     def warning(self, msg):
-        self.logline("WARNING:" + msg, red=True)
+        if self.verbosity >= -1:
+            self.logline("WARNING:" + msg, red=True)
 
     def error(self, msg):
-        self.logline("ERROR: " + msg, red=True)
+        if self.verbosity >= -1:
+            self.logline("ERROR: " + msg, red=True)
```

Comment:
Not sure if we should silence error messages...

Reply:
So this would be if someone does `tox -qq` in other words, really really quiet `tox`. I think in that case it's fine. If someone does `tox -q` they still see ERROR messages. Make sense?